docs/topics/proc_viewing-the-booster-source-code-and-readme.adoc

```diff
@@ -10,11 +11,10 @@ One of the following:
 
 .Procedure
 
-. Use the {name-launcher} tool to generate your own version of the booster.
-. View the generated GitHub repository or download and extract the ZIP file that contains the booster source code.
+. Use the {name-launcher} tool to generate your own version of the example.
+. View the generated GitHub repository or download and extract the ZIP file that contains the example source code.
```

Comment:
the example source code--> the example application source code

Reply:
Yes. Thanks again.